### PR TITLE
Toc styles

### LINKED ISSUE #169: New Basic TOC styles
image

ul.toc-list {
list-style-type: none;
padding-inline: 0px;
}

ul.toc-sub-list {
list-style-type: none;
padding-inline: 24px;
}

li.toc-h2 {
   font-weight: 700;
}   

li.toc-h3 {
margin: 16px 0;
}

### PR DESCRIPTION
**Pull request by sforschen**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #169 

Test URLs:
- Before: https://main--elixirsolutions--hlxsites.hlx.live/
- After: https://toc-styles--elixirsolutions--hlxsites.hlx.live/

### CODE REVIEW

**Comment by sforschen** [comment]
found on the terms of service page

**Comment by janefrommaine** [comment]
@sforschen, looking good!
There are somethings we can do to improve accessibility but not sure if you want to tackle that in a future ticket.
I think these were things we discussed for mobile we'd want the TOC to go above the content. And maybe we might find that the TOC should go to the left of the content in desktop. Maybe we should review with Sarah K. at some point.

Those seem out of scope for this ticket. Just want to confirm with you before I approve.

**Comment by sforschen** [comment]
@janefrommaine Because we have that column conundrum issue I would see the toc as part of that, the layout is not ideal and we talked about making it sticky or some other fix. I think the scope will be a lot bigger when we get into updating how we handle sidebars and columns. For this ticket, I just want to make it not bulleted so its useable in the meantime.

**Review by janefrommaine — APPROVED** [review]

**Pull request merged**
